urllib3/util/connection.py

```diff
@@ -1,16 +1,19 @@
 from __future__ import absolute_import
 import socket
+from urllib3.util.rfc6555 import (
+    happy_eyeballs_algorithm
+)
 from .wait import wait_for_read
 from .selectors import HAS_SELECT, SelectorError
 
+ENABLE_HAPPY_EYEBALLS = True
```

Comment:
This doesn't seem to get used anywhere: it's just a global variable that stays True all the time. What's it for?